config/environment.js

```diff
@@ -41,11 +64,16 @@ module.exports = function (environment) {
 
     ENV.APP.rootElement = '#ember-testing';
     ENV.APP.autoboot = false;
+    ENV['ember-simple-auth'] = {
+      routeAfterAuthentication: 'points'
+    }
```

Comment:
two entries of the same

Reply:
I am getting route issue (same as mentioned above) in login acceptance test too, so while exploring about solutions i have added this